### PR TITLE
Add migration script that only relies on v2

### LINKED ISSUE #42: Improve migration from v1 to v2
Migrating the entire v1 database schema is probably too much effort. Instead, it might be enough to export subscriptions with v1 and import the OPML export in v2. Problems with this first alternative:

- User must have both v1 and v2 installed (or make an export *before* upgrading to v2)
- v2 database must be at different location from v1 (that's not the case in the default config)

Second alternative: A python script (preferably without dependencies) that renames the v1 db, exports subscriptions, and re-imports them with v2. This would solve the problems of the first alternative, but is more work.

I'd love to see everyone's thoughts on this.

### PR DESCRIPTION
**Pull request by woefe**
This migration script relies only on v2 being installed and available as Python import. It reads subscriptions and videos directly from the v1 SQL database and adds them to a v2 database. The duration column in v2 is always set to zero.

This is a possible replacement for PR #50 and fixes #42

### CODE REVIEW

**Comment by woefe** [comment]
@EmRowlands Can you try this migration script on your setup and also let me know if it works okay?

File: scripts/migrate.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+#  along with ytcc.  If not, see <http://www.gnu.org/licenses/>.
+
+import sqlite3
+import sys
+import textwrap
+from pathlib import Path
+
+
+def error(message: str):
+    print("Error:", message)
```
Comment (EmRowlands) [inline scripts/migrate.py:28]: I suggest sending errors to `stderr`
```suggestion
    print("Error:", message, file=sys.stderr)
```

**Review by EmRowlands — CHANGES_REQUESTED** [review]
I think the only thing I would suggest is sending errors to `stderr`. Other than that, this looks good! I have tested it and it works for me, producing an identical database (except with different `videoid`s) to the one I created.

**Comment by woefe** [comment]
@EmRowlands Thank you very much for your feedback and your help on this PR! ❤ And sorry that that I didn't merge your efforts from #50.

**Comment by EmRowlands** [comment]
Hey, no need to apologise. I use ytcc pretty much every day and I'm happy to help in any way I can. Be it writing code, reviewing code, or just submitting bugs.

**Pull request merged**